xapian-bindings/python3/.gitignore

```diff
@@ -6,9 +6,13 @@
 /.libs
 /*.pyc
 /_xapian.la
+/_xapianletor.la
 /xapian
+/xapianletor
 /modern
 /except.i
+/exceptletor.i
+/database_two_docs
```

Comment:
Is there a reason not to prefix this `db_` (instead of `database_`) so it matches other test databases? Would also mean it's covered by the local clean rule.

Reply:
fixed in a fixup commit.